modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaJSImportMapsContributorCheck.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.check.util.SourceUtil;
+
+import java.util.List;
+
+/**
+ * @author Bryce Osterhaus
+ */
+public class JavaJSImportMapsContributorCheck extends BaseFileCheck {
+
+	@Override
+	public boolean isLiferaySourceCheck() {
+		return true;
+	}
+
+	@Override
+	protected String doProcess(
+		String fileName, String absolutePath, String content) {
+
+		if (fileName.endsWith("JSImportMapsContributor.java")) {
```

Comment:
We will have `DynamicJSImportMapsContributor` too after this PR -> https://github.com/liferay-frontend/liferay-portal/pull/4884 but luckily we are covered :four_leaf_clover: